pkgs/development/idris2-modules/with-packages.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, makeWrapper, symlinkJoin, callPackage, idris2, ... }: packages:
+
+let
+  name = idris2.name;
+  paths = lib.closePropagation packages;
+in symlinkJoin {
```

Comment:
In  `idris.with-packages`, they use `lib.appendToName "with-packages"` here. Unfortunately, running `nix run .#idris2` will fail because it would expect a `idris2-with-packages` binary, which we don't have.

Idris has the same problem actually (`nix run .#idris` results in an error). Not sure if they know.